en/docs/develop-components/develop-a-webhook.md

```diff
@@ -12,39 +12,37 @@ In this tutorial, you will address the requirement by doing the following:
 
 ## Prerequisites
 
-Before you try out the steps in this guide, complete the following:
+Before you begin:
 
- - If you are signing in to the Choreo Console for the first time, create an organization as follows:
-    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/), and sign in using your preferred method.
-    2. Enter a unique organization name. For example, `Stark Industries`.
+1. **Create a Choreo Organization**:
```

Comment:
We generally don't use bold formatting and title caps in step-by-step instructions. See https://developers.google.com/style/text-formatting and https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting#text-formatting-guidelines.